safe/gui/tools/help/options_help.py

```diff
@@ -208,7 +208,7 @@ def content():
     bullets.add(m.Text(
         m.ImportantText(tr(
             'Contact email')),
-        tr(' - Use this option to specify the contact person\s email '
+        tr(' - Use this option to specify the contact person\'s email '
```

Comment:
In case like this, I think I would prefer for you to use double quote so you don't have to escape the quote inside.